lola.html

```diff
@@ -905,31 +1012,101 @@ <h2>Followup Work</h2>
+      </p>
+
+      <p>
+        Load management: it is possible that the source server issuing Move activities could cause many follower
+        agents to issue a great deal of data fetching requests to the destination.  A malicious source server
+        could issue false Move Activities in an attempt to overload the alleged destination server with follow
+        requests and other requests. More discussion of this threat is welcome, but so far this seems self-limiting
+        because a malicious source server would have to have followers to send to a false destination, and
+        would presumably lose its followers and run out of them during this process.
+      </p>
+
+      <p>
+        Open Issue: Can the destination server issue a Move activity if necessary?  It cannot be trusted, because a 
+        malicious actor could issue a Move activity for an account they have no legitimate access to.  It's not enough 
+        to check the source Actor to see if it has a movedTo value, because the malicious actor could specifically 
+        target accounts whose Actor has disappeared.  This is another good reason for source servers to host moved 
+        Actors as long as possible - to reduce opportunities for fraudulent Move Actor notifications. Recipients of 
+        Move Actor activities from any other than the server hosting the original Actor MAY ignore these Move requests 
+        if they cannot be verified.
       </p>
+      
 
       <h4>Hosting Moved Actors</h4>
 
       <p>
-        The source server SHOULD offer UI for the user to choose to mark their Actor as moved to the new server.   This is accomplished via the ‘movedTo’ property as explained and described in Data Portability in ActivityPub [TODO: validate that the required value for movedTo is a single URL value pointing ideally to an Actor elsewhere]. The source server MUST offer UI for the user to delete their content (consider the “right to be forgotten” if the user no longer trusts the source server with their content) while still hosting just the Actor with the ‘movedTo’ property.  The source server SHOULD attempt to maintain this information for at least a year even if the account is otherwise shut down (although this would of course not be maintained if the server changes domains, loses state, or has another significant event to break the continuity of redirect from old to new actor.)
+        The source server SHOULD offer UI for the user to choose to mark their Actor as moved to the new server.   
+        This is accomplished via the ‘movedTo’ property as explained and described in Data Portability in ActivityPub 
+        [TODO: validate that the required value for movedTo is a single URL value pointing ideally to an Actor elsewhere]. 
+        The source server MUST offer UI for the user to delete their content (consider the “right to be forgotten” if the 
```

Comment:
```suggestion
        The source server MUST offer an affordance (such as a dedicated UI exposed to the user) to delete their content (consider the “right to be forgotten” if the 
```

Reply:
Note that if you're using "right to be forgotten" in the GDPR sense, this would mean effectively removing the data from public _and admin_ access, but also notifying other servers to do the same (send out an Update maybe?), and also keeping some kind of receipt of the user intent and timestamp of deletion process. Might be out of scope of this specific doc but worth writing a followup FEP, since it's debatably in scope of the task force?